content/docs/gke/deploy/minikf-gcp.md

```diff
@@ -0,0 +1,93 @@
++++
+title = "Deploy using MiniKF on GCP"
+description = "Instructions for using the Google Cloud Marketplace to deploy MiniKF (mini Kubeflow) on Google Cloud Platform (GCP)."
+weight = 4
++++
+
+This guide describes how to launch a MiniKF VM on GCP. MiniKF is a
+production-ready, full-fledged Kubeflow deployment that installs in minutes.
+It's a fast and easy way to get started with Kubeflow. With just a few clicks,
+you are up for experimentation, and for running complete Kubeflow Pipelines.
+
+MiniKF installs Kubeflow 0.7.0 and supports NVIDIA GPUs.
+
+### Installing MiniKF on GCP
```

Comment:
I think we need a "Before you start" section, similar to this one:
https://www.kubeflow.org/docs/gke/deploy/deploy-ui/#before-you-start

The reason is that, as a reader, I'm not sure how much of the other stuff I need to do before I can install MiniKF. Do I need to set up a GCP project? (Yes) Does my project need the same setup as described for a [GKE deployment of Kubeflow](https://www.kubeflow.org/docs/gke/deploy/project-setup/)? (Not sure). Do I need to set up [OAuth](https://www.kubeflow.org/docs/gke/deploy/oauth-setup/)? (don't think so, but it'd be nice to know for sure).